sdk/resources/Microsoft.Azure.Management.Resource/src/Generated/Models/TemplateLink.cs

```diff
@@ -33,17 +33,22 @@ public TemplateLink()
         /// uri or id property, but not both.</param>
         /// <param name="id">The resource id of a Template Spec. Use either the
         /// id or uri property, but not both.</param>
-        /// <param name="relativePath">Applicable only if this template link
-        /// references a Template Spec. This relativePath property can
-        /// optionally be used to reference a Template Spec artifact by
-        /// path.</param>
+        /// <param name="relativePath">The relativePath property can be used to
+        /// deploy a linked template at a location relative to the parent. If
+        /// the parent template was linked with a TemplateSpec, this will
+        /// reference an artifact in the TemplateSpec.  If the parent was
+        /// linked with a URI, the child deployment will be a combination of
+        /// the parent and relativePath URIs</param>
+        /// <param name="queryString">The query string (for example, a SAS
+        /// token) to be used with the templateLink URI.</param>
         /// <param name="contentVersion">If included, must match the
         /// ContentVersion in the template.</param>
-        public TemplateLink(string uri = default(string), string id = default(string), string relativePath = default(string), string contentVersion = default(string))
```

Comment:
So, this is not breaking in the sense things won't compile, but it is breaking in the sense current code will pass in queryString for what they thought was contentVersion possibly

Reply:
Ok. So how do we go about this? And since this is using a new API version altogether, does that affect things?

Reply:
Created a PR to update the parameter ordering in swagger https://github.com/Azure/azure-rest-api-specs/pull/11883

Reply:
PR is merged. I will regenerate and push in a fix tomorrow.